src/components/Regions.css

```diff
@@ -0,0 +1,44 @@
+/* .Regions_container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+} */
+
```

Comment:
- [ ] Kindly remove all Commented code so that your file may be clean

Reply:
sure